src/tarantool/bank.clj

```diff
@@ -37,10 +37,13 @@
                              balance INT NOT NULL)")])
            (doseq [a (:accounts test)]
                (info "Populating account")
-               (sql/insert! conn table-name {:id      a
-                                             :balance (if (= a (first (:accounts test)))
-                                                       (:total-amount test)
-                                                       0)}))))
+               ; Distribute initial balances uniformly.
+               (let [initial-balance (/ (:total-amount test)
```

Comment:
I would rename `initial-balance` to `initial-balance-per-account`.

Reply:
It feels a bit too long for me, but okay. Changed.